net-p2p/bitcoin-qt/bitcoin-qt-0.15.1.ebuild

```diff
@@ -0,0 +1,246 @@
+# Copyright 2010-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DB_VER="4.8"
+inherit autotools bash-completion-r1 db-use gnome2-utils kde4-functions xdg-utils
```

Comment:
what's `kde4-functions` still doing there?

Reply:
For `buildsycoca` - do we not need that anymore? Not entirely sure what it does...

Reply:
not in a long time, kded is doing that as-needed. the only thing left in `buildsycoca` is a permissions check.

Reply:
Okay, dropped it.